web/src/algorithms/MeteorAlgorithm.ts

```diff
@@ -1,16 +1,34 @@
-import { neo } from "config"
-import { randomInt } from "crypto"
-import { QueryResult } from "neo4j-driver"
+import { Neo4jModule } from '@/config/neo4j/neo4j.module';
```

Comment:
Questo cambiamento potevi anche non farlo, secondo me era più leggibile prima

Reply:
così ogni db ha un proprio modulo che lavora in modo asincrono rispetto all'altro, per questo ci mette anche meno tempo nell'eseguire una richiesta